sumatra/recordstore/django_store/__init__.py

```diff
@@ -279,8 +279,8 @@ def clear(self):
         if not db_config.configured:
             db_config.configure()
         #management.call_command('sqlclear', 'django_store', database=self._db_label)  # this produces coloured output, need no_color option from Django 1.7
-        cmds = ["BEGIN;"] + ['DROP TABLE "django_store_{}";'.format(x)
-                             for x in ("record", "record_output_data", "record_input_data", "record_dependencies",
```

Comment:
Removing `django_store_record_output_data` indeed fixes the django error that I saw. Just to make sure I understand the change, what are the tables that should be dropped at this point and why should `django_store_record_output_data` not be one of them?